lib/reform/form/multi_parameter_attributes.rb

```diff
@@ -24,10 +24,12 @@ def call(params)
 
   private
     def params_to_date(year, month, day, hour, minute)
-      return nil if [year, month, day].any?(&:blank?)
+      if [year, month, day].any?(&:blank?) || !Date.valid_date?(*[year, month, day].map(&:to_i))
```

Comment:
Blank strings become `0` when coerced via `to_i`. Given that any `0` is understood as an invalid date, what do you think of skipping the `blank?` check? Maybe I'm missing something?